payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/MandateTextElement.kt

```diff
@@ -0,0 +1,17 @@
+package com.stripe.android.ui.core.elements
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.ui.core.forms.FormFieldEntry
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class MandateTextElement(
+    override val identifier: IdentifierSpec,
+    val stringResId: Int,
+    val merchantName: String?,
+    override val controller: InputController? = null,
+) : FormElement() {
```

Comment:
Why isn't this StaticTextElement?

Reply:
Wanted to keep the UI of static text and mandate text different. It makes it so creating forms doesn't require you to have any knowledge of UI. Previously you had to add some font styling to the spec.